src/Voron/Impl/Paging/AbstractPager.cs

```diff
@@ -248,14 +248,14 @@ public static bool IsKeySizeValid(int keySize)
             return true;
         }
 
-        protected List<Win32MemoryMapNativeMethods.WIN32_MEMORY_RANGE_ENTRY> SortedPagesToList(
-            List<TreePage> sortedPages)
+        protected List<Win32MemoryMapNativeMethods.WIN32_MEMORY_RANGE_ENTRY> SortedPagesToList(List<TreePage> sortedPages)
         {
             var rangesList = new List<Win32MemoryMapNativeMethods.WIN32_MEMORY_RANGE_ENTRY>();
 
             long lastPage = -1;
-            const int numberOfPagesInBatch = 8;
-            var sizeInPages = numberOfPagesInBatch; // OS uses 32K when you touch a page, let us reuse this
+            int numberOfPagesInBatch = Sparrow.Platform.RunningOnPosix ? 32 : 8; // 128k in linux, 32k in windows - when you touch a page, let us reuse this
```

Comment:
Where did you get the 128K on linux? 
Also, don't assume that you know what the page size is, we'll likely change that a few times.